packages/matchbox/src/components/Banner/styles.js

```diff
@@ -71,6 +71,9 @@ export function dismissBase() {
   return `
     padding: 0.25rem;
     transition: background ${tokens.motionDuration_fast} ${tokens.motionEase_in_out};
+    &:hover {
```

Comment:
Why is this style prop only set on hover?

Reply:
No idea, I've always done it because it feels more semantically correct. But either way don't think it matters too much